src/main/java/nl/dtls/fairdatapoint/api/config/RestApiContext.java

```diff
@@ -249,11 +249,15 @@ private Repository getBlazeGraphRepository() {
     private Repository getGraphRepository() {
 
         Repository repository = null;
-        if (!graphDbUrl.isEmpty() && !graphDbRepository.isEmpty()) {
-            LOGGER.info("Initializing graphDB repository");
-            RepositoryManager repositoryManager = new RemoteRepositoryManager(graphDbUrl);
-            repositoryManager.initialize();
-            repository = repositoryManager.getRepository(graphDbRepository);
+        try {
+            if (!graphDbUrl.isEmpty() && !graphDbRepository.isEmpty()) {
+                LOGGER.info("Initializing graphDB repository");
+                RepositoryManager repositoryManager = new RemoteRepositoryManager(graphDbUrl);
+                repositoryManager.initialize();
+                repository = repositoryManager.getRepository(graphDbRepository);
+            }
+        } catch(Exception e) {
+            LOGGER.error("Initializing graphDB repository");

```

Comment:
Catching `Exception` is very broad, are there any more specific exceptions we can catch here?